public-api/src/main/java/org/pmiops/workbench/service/DomainInfoService.java

```diff
@@ -31,4 +33,25 @@ public DomainInfoService(DomainInfoDao domainInfoDao, DomainMapper domainMapper)
                 .map(domainMapper::dbModelToClient)
                 .collect(Collectors.toList());
     }
+
+    public List<Integer> getTestOrderFilter(int testFilter, int orderFilter) {
+        Integer getTests = null;
+        Integer getOrders = null;
+
+        if (testFilter == 1 && orderFilter == 1) {
+            getTests = 1;
+            getOrders = 0;
+        } else if (testFilter == 1 && orderFilter == 0) {
+            getTests = 1;
+            getOrders = 2;
+        } else if (testFilter == 0 && orderFilter == 1) {
+            getTests = 2;
+            getOrders = 0;
+        } else if (testFilter == 0 && orderFilter == 0) {
+            getTests = 2;
+            getOrders = 2;
+        }
```

Comment:
nit: what does testFilter 0 or 1 represent? orderFilter 0 or 1? can we use a constant instead of a magic number?

Reply:
Added enum in here.

Reply:
Changed this. Thanks.